Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,33 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            Console.WriteLine("What's your name?");
+            string name = Console.ReadLine();         //NAME
+            Console.WriteLine("What's your surname?");
+            string surname = Console.ReadLine();      //SURNAME
+            Console.Write("How old are you?");
+            var age = Console.ReadLine();             //AGE
+            Console.Write("How much do you weight?"); 
+            double weight = Convert.ToDouble(Console.ReadLine());
+            Console.Write("How high are you (in meters)?");  //HEIGH
+            double height = Convert.ToDouble(Console.ReadLine());
+            Console.WriteLine(name+" "+surname+" is "+age+" years old, weight "+weight+" kg and is "+height+" m tall.");   //PRINTING INFO
+            Console.WriteLine("His BMI equals: "+ weight /(height*height)+".");   //BMI
+
+            //ANOTHER PERSON
```

Comment:
Well done for separating the two people with a comment. However, blank lines before and after are not needed. you can remove the blank below and keep the one above, so that the comment actually stays right next to another person's info.